lastseen.py

```diff
@@ -0,0 +1,46 @@
+import re
+from datetime import datetime
+from errbot import BotPlugin, botcmd, re_botcmd
+
+
```

Comment:
@JakubNvk are these two empty lines necessary in order for `PEP8` checks to pass? If not, please feel free to remove them.